apps/modernization-ui/src/apps/search/advancedSearch/AdvancedSearch.scss

```diff
@@ -1,5 +1,17 @@
 @use 'styles/colors';
 
+.advanced-search-title {
+    font-size: 100%;
+}
+
+.advanced-search-results-title {
+    margin-block-start: 0 !important;
+    margin-block-end: 0 !important;
+    margin-inline-start: 0 !important;
+    margin-inline-end: 0 !important;
+    font-size: 1.13rem;
+}
+
 button {
```

Comment:
![Screenshot 2024-01-30 at 12 42 37 PM](https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/d80eb19a-a4d3-4e7d-88ab-8aca47239a57)
Needed to prevent this

Reply:
I think a `margin: 0` would also fix the issue.  The real culprit is the chips, it should have a `height: fit-content`